src/lightning_app/utilities/packaging/lightning_utils.py

```diff
@@ -113,13 +113,15 @@ def _prepare_lightning_wheels_and_requirements(root: Path) -> Optional[Callable]
         # building and copying launcher wheel if installed in editable mode
         launcher_project_path = get_dist_path_if_editable_install("lightning_launcher")
         if launcher_project_path:
+            logger.info("Packaged Lightning Launcher with your application.")
```

Comment:
noice :))

Reply:
ahahah yeah, I never remembered if I needed to put a 0 or 1 ;)